WORKSPACE

```diff
@@ -177,7 +171,7 @@ jvm_maven_import_external(
     name = "org_typelevel__cats_core",
     artifact = scala_mvn_artifact(
         "org.typelevel:cats-core:0.9.0",
-        default_scala_major_version(),
+        SCALA_MAJOR_VERSION,
```

Comment:
Why the change from method to const?

Reply:
Is there a reason to wrap the const inside a function? The default version functions are removed, ant the stored version should be a single source of truth.